app/v2/models/request/amendHistoricFhlUkPiePeriodSummary/AmendHistoricFhlUkPiePeriodSummaryRawData.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.request.amendHistoricFhlUkPiePeriodSummary
+
+import play.api.libs.functional.syntax.{ toFunctionalBuilderOps, unlift }
+import play.api.libs.json.{ JsPath, JsValue, OWrites }
+import v2.models.request.RawData
+
+case class AmendHistoricFhlUkPiePeriodSummaryRawData(nino: String, periodId: String, body: JsValue) extends RawData
+
+object AmendHistoricFhlUkPiePeriodSummaryRawData {
+  implicit val writes: OWrites[AmendHistoricFhlUkPiePeriodSummaryRawData] = (
+    (JsPath \ "nino").write[String] and
+      (JsPath \ "periodId").write[String] and
+      (JsPath \ "request").write[JsValue]
+  )(unlift(AmendHistoricFhlUkPiePeriodSummaryRawData.unapply))
+}
```

Comment:
This `Writes` might not be needed. Existing endpoints use a `Writes` here for auditing but the audit folks seem to be wanting a different (partially flattened) format for the request JSON for newer endpoints.